mythril/laser/ethereum/call.py

```diff
@@ -63,6 +63,7 @@ def get_callee_address(global_state:GlobalState, dynamic_loader: DynLoader, symb
         # attempt to read the contract address from instance storage
         try:
             callee_address = dynamic_loader.read_storage(environment.active_account.address, index)
+        # TODO: verify whether this happens or not
         except:
```

Comment:
There is still a very broad exception catch here

Reply:
It seems there are atleast 5-6 exceptions possible to be raised at for this catch, so how about shifting this broad exception to read_storage function which raises some sort of ErrorReadingStorage exception when it catches any exception?